Polyhedron/demo/Polyhedron/Polyhedron_demo.cpp

```diff
@@ -59,12 +92,21 @@ Polyhedron_demo::Polyhedron_demo(int& argc, char **argv,
   QCommandLineOption verbose("verbose",
                                    tr("Print the paths explored byt the application searching for plugins."));
   parser.addOption(verbose);
+  QCommandLineOption old("old",
+    tr("Force OpenGL 2.1 context."));
+  parser.addOption(old);
   parser.addPositionalArgument("files", tr("Files to open"), "[files...]");
   parser.process(*this);
   d_ptr->mainWindow.reset(new MainWindow(parser.isSet(verbose)));
   MainWindow& mainWindow = *d_ptr->mainWindow;
   
   mainWindow.setWindowTitle(main_window_title);
+
+  //find a suitable context
+  if (OGLSupports(4, 3, false, QSurfaceFormat::CoreProfile))
+  {
+    
+  }
```

Comment:
@maxGimeno Is that normal that the body of the `if` is empty? I thought you wanted to set the default format to 4.3 core profile.

Reply:
Nope, I forgot to remove it.